libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.m

```diff
@@ -241,6 +240,7 @@ - (SFSyncState*) syncDownWithTarget:(SFSyncDownTarget*)target soupName:(NSString
  */
 - (SFSyncState*) syncDownWithTarget:(SFSyncDownTarget*)target options:(SFSyncOptions*)options soupName:(NSString*)soupName updateBlock:(SFSyncSyncManagerUpdateBlock)updateBlock {
     SFSyncState* sync = [SFSyncState newSyncDownWithOptions:options target:target soupName:soupName store:self.store];
+    [self logAnalyticsEventWithSyncState:sync name:@"syncDown"];
```

Comment:
The sync state will not be fully populated until the sync is running (e.g. the number of records is only known once we go to the server).
So this needs to move to here: https://github.com/bhariharan/SalesforceMobileSDK-iOS/blob/248aa0038fad2c55ab21f6baf187b9a71baa8ff5/libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.m#L198